app/services/loader.js

```diff
@@ -13,7 +13,7 @@ export default Service.extend({
     if (config.isExternal) {
       options.url = url;
     } else {
-      options.url = config.withoutPrefix ? `${adapter.host}/${url}` : `${adapter.urlPrefix()}${url}`;
+      options.url = config.withoutPrefix ? `${adapter.host}${url}` : `${adapter.urlPrefix()}${url}`;
```

Comment:
@niranjan94 But the thing is ```${adapter.host}${url}``` and ```${adapter.urlPrefix()}${url}``` give different output. If I do the check you told above, for the first post request which gives task_url(which also uses loader and has withoutPrefix set to null), has a url open-event-api.herokuapp.comevents/553fd33/export/json.